CMakeLists.txt

```diff
@@ -254,21 +254,17 @@ else()
     set(PROGNAME keepassxc)
 endif()
 
-if(APPLE AND WITH_APP_BUNDLE AND "${CMAKE_INSTALL_PREFIX}" STREQUAL "/usr/local")
-    set(CMAKE_INSTALL_PREFIX "/Applications")
```

Comment:
You need to set the CMAKE_INSTALL_PREFIX to /Applications. It is inappropriate for the CMakeFile to override this setting.

Reply:
@droidmonkey I actually think it is a good idea to remove `make install` to `/Applications`. Since Mojave it updates the application fingerprint and auto-type stops working until I manually uncheck and check the app bundle again on accessibility permissions.